### DISCUSSION TITLE
Dedicated "Password Generator" feature

### DISCUSSION THREAD

**Discussion by 97ryan** [Kunzisoft/KeePassDX#1391] [Ideas]
It would be awesome if there was a section in the app for a password generator, like KeePassXC has. As it stands, I have to go in to create entries to generate one, and that just seems like wasted effort if I don't end up making a new entry anyway. Sometimes, I just want to generate something.

Bitwarden also has this option too as a menu inside the app. I have read your previous suggestions to include this elsewhere (such as keyboard or in launcher) but having the option under the "+" menu, or somewhere inside the app would really be nice for a lot of people.

**Comment by J-Jamet** [comment]
Linked to https://github.com/Kunzisoft/KeePassDX/issues/1234

**Comment by ghost** [comment]
It has the advantage though that should something go wrong with your clipboard you still have a copy in your database. You probably know what you're doing but other users might misuse this feature and end up with lost passwords.
